beta/nncf/tensorflow/pruning/base_algorithm.py

```diff
@@ -162,6 +157,19 @@ def get_transformation_layout(self, model: tf.keras.Model) -> TFTransformationLa
 
         return transformations
 
+    def _get_insertion_command_binary_mask(self, layer_name, attr_name):
+        op_name = layer_name + '_' +  attr_name + OP_NAME_BM
```

Comment:
Will it work if I apply two pruning algorithms to the model?

Reply:
It'll raise the error